tools/omfwd.c

```diff
@@ -885,12 +895,26 @@ static rsRetVal doTryResume(wrkrInstanceData_t *pWrkrData)
 				pData->target, pData->port, gai_strerror(iErr));
 			ABORT_FINALIZE(RS_RET_SUSPENDED);
 		}
+		address = pData->target;
+		if(pData->address) {
+			/* The AF of the bind addr must match that of target */
+			hints.ai_family = res->ai_family;
+			hints.ai_flags |= AI_PASSIVE;
+			if((iErr = getaddrinfo(pData->address, pData->port, &hints, &addr)) != 0) {
+				errmsg.LogError(0, RS_RET_SUSPENDED,
```

Comment:
you can drop the ```errmsg.``` prefix. We are currently in a project to gradually phase the old method out. Not a hard request, though.

Reply:
Done